handbook/engineering/index.md

```diff
@@ -35,6 +35,15 @@
   - [Adding and debugging ping data](adding_ping_data.md)
   - [Adding buildkite secrets](adding_buildkite_secrets.md)
 
+## Guiding principles
+
+We build things in this order:
+
+1. **Make it work**: Build the minimal useful thing that we can use to start collecting feedback and validating we are on the right track. Take shortcuts where possible because this work might get thrown away if we discover we are going in the wrong direction.
```

Comment:
```suggestion
1. **Make it work**: Build the minimal useful thing that we can use to start collecting feedback and validating we are on the right track. Take shortcuts where possible (cut scope, not quality) because this work might get thrown away if we discover we are going in the wrong direction.
```